docs/testing.rst

```diff
@@ -1,7 +1,22 @@
-Test structure
-====================
+Tests
+========
 
-The tests in TRD are run in the following order:
+Test Execution
+---------------
+To run the tests locally you need to have `pytest <https://pypi.org/project/pytest/>`_ installed:
+::
+
+    pip install pytest
```

Comment:
Should we split the requirements.txt in two files where one is for users and the other for devs? Then add pytest there as well and install everything through the file? [See comment here already](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/master/requirements.txt)